Mathlib/Algebra/Group/Commute/Defs.lean

```diff
@@ -44,6 +44,9 @@ section Mul
 
 variable {S : Type*} [Mul S]
 
+@[to_additive]
+theorem _root_.commute_def {a b : S} : Commute a b ↔ a * b = b * a := .rfl
```

Comment:
This lemma got merged in-between: [commute_iff_eq](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Group/Commute/Defs.html#commute_iff_eq)